tests/drizzle/test_drizzle.py

```diff
@@ -0,0 +1,37 @@
+import numpy as np
+from astropy import wcs
+
+from drizzlepac import cdriz
+
+def test_zero_input_weight():
```

Comment:
I tried running this test on the master branch and it passed there too.  Did I make a mistake, or does the test need adjustment so that it covers the problem?

Reply:
No mistake. The test was bad.